plugins/bnr-dp/fu-bnr-dp-device.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2024-2025 B&R Industrial Automation GmbH
```

Comment:
just the start date in copyright please -- same for all the files.

Reply:
That's up to you; I think copyright normally starts on creation but I don't mind either.